frontend/tests/features/wheresMyGene/cellTooltip.test.ts

```diff
@@ -10,12 +9,6 @@ import {
 const { describe } = test;
 
 describe("cell tooltip", () => {
-  /**
-   * TODO(thuang): Remove forceRun when all WMG e2e tests are enabled.
-   * `forceRun` is just to incrementally add tests back in the meantime
-   */
-  conditionallyRunTests({ forceRun: true });
```

Comment:
now that we've added back all WMG tests, we can remove this `conditionallyRunTests` block. Here and throughout